src/components/UpdateVotes.jsx

```diff
@@ -0,0 +1,41 @@
+import { patchVotes } from "../../api";
+import {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+import '../../styling/Vote.css'
+
+export default function UpdateVotes(){
+    const {article_id} = useParams()
+    const [votesCount, setVotesCount] = useState(0)
+    const [err, setErr] = useState(null)
+
```

Comment:
These two functions (`handleVotesClick` and `handleDeleteVotesClick`) are very similar, could you refactor them into one function which takes an argument?